react/features/user-customization/actions.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import { getLogger } from 'jitsi-meet-logger';
+
+import { doGetJSON } from '../base/util';
+
+import { SET_USER_CUSTOM_DATA } from './actionTypes';
+import { extractFqnFromPath } from './functions';
+
+const logger = getLogger(__filename);
+
+/**
+ * Fetches custom branding data.
+ * If there is no data or the request fails, sets the `customizationReady` flag
+ * so the defaults can be displayed.
+ *
+ * @returns {Function}
+ */
+export function fetchCustomBrandingData() {
+    return async function(dispatch: Function, getState: Function) {
+        const state = getState();
+        const baseUrl = state['features/base/config']._brandingDataUrl;
```

Comment:
Why is this a "private" config option? Why is it not documented?

Reply:
Made it public. Unfortunately I have not found a single external url documented. I have added this to the list of other undocumented urls.